docs/src/components/Theming/TokenList.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { useTheme } from '@aws-amplify/ui-react';
+import { isDesignToken } from '@aws-amplify/ui';
+import {
+  SpaceBlock,
+  BorderWidthBlock,
+  FontFamilyBlock,
+  FontSizeBlock,
+  LineHeightBlock,
+  FontWeightBlock,
+  ColorBlock,
+  RadiusBlock,
+} from './TokenBlocks';
+
+type Namespaces =
+  | 'colors'
+  | 'radii'
+  | 'space'
+  | 'borderWidths'
+  | 'lineHeights'
+  | 'fonts'
+  | 'fontSizes'
+  | 'fontWeights';
+
+export function createTokenList(tokens) {
+  // Creates a flattened array out of the token object passed to createTokenList()
+  let tokenList = [];
+  function iterateGroup(group) {
+    Object.values(group).forEach((value) => {
+      if (isDesignToken(value)) {
+        tokenList.push({
+          ...value,
+        });
+      } else {
+        iterateGroup(value);
+      }
+    });
+  }
+  if (isDesignToken(tokens)) {
+    tokenList.push(tokens);
+  } else {
+    iterateGroup(tokens);
+  }
+  return tokenList;
+}
+
+type TokenItemProps = {
+  variation: Namespaces;
+  children: React.ReactNode;
+};
+
+export function TokenItem({ variation, children }: TokenItemProps) {
+  return (
+    <li>
+      <div className={`docs-tokenItem docs-tokenItem--${variation}`}>
```

Comment:
Would it make sense to just move the `className` to the `li` and remove this `div`? Just asking out of curiosity

Reply:
Yeah that makes sense, we don't need the extra div. Updated  in latest.